presto-hive/src/main/java/com/facebook/presto/hive/auth/KerberosConnectorModule.java

```diff
@@ -35,7 +36,7 @@
 
 import static com.google.common.base.MoreObjects.firstNonNull;
 
-public class HdfsAuthenticatingConnectorModule
+public class KerberosConnectorModule
```

Comment:
For the sake of code reusablility i would leave only one single module, say `KerberosConnectorModule`. Than based on the `HiveClientConfig` i would create just different implementations of the `HadoopAuthentication` in `getHadoopAuthentication` method.